dts/bindings/i3c/i3c-device.yaml

```diff
@@ -61,3 +61,9 @@ properties:
     type: int
     description: |
       Dynamic address to be assigned to the device.
+
+  supports-setaasa:
+    type: boolean
+    description: |
+      Indicates if the device supports the CCC SETAASA. If true, it will
+      be used as an optimization for bus initialization.
```

Comment:
nit: if more _supports-*_ are required in the future, a bit field could be more future-proof, but not sure if it can be the case.

Reply:
Yeah... it definitely could and that was something was thinking about, but that could come when more "supports" are implemented rather than using an inefficient bool for each of them.